README.md

```diff
@@ -17,8 +17,10 @@ For a development install:
     conda create -n jupyter-vcdat -c cdat/label/nightly -c conda-forge nodejs "python>3" vcs jupyterlab pip nb_conda nb_conda_kernels plumbum
     source activate jupyter-vcdat
 
+    # Install azy_import
```

Comment:
# Install azy_import -> # Install lazy_import
Minor comment typo there

Reply:
fixed

Reply:
Looks good! Should we wait for Sterling or William to review before merging?